app/main.py

```diff
@@ -1,8 +1,19 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int, ) -> None:
+        self.name = name
+        self.age = age
+        self.people[name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    result = [Person(person["name"], person["age"]) for person in people]
+    for i in range(len(people)):
+        for person in result:
+            if "husband" in people[i] and people[i]["husband"] == person.name:
```

Comment:
Consider using `people[i].get('husband')` instead of explicitly checking for the presence of the 'husband' key. This approach simplifies the code and avoids potential `KeyError` exceptions, making it cleaner and more concise as per the checklist recommendation.